### PR TITLE
nullsafety version

### PR DESCRIPTION
**Pull request by jWinterDay**
In this PR I migrated package sdk version to nullsafety (and for example)
Also I regenerated native part for ios and android (example/ios, example/android)

This was tested only for example and should more tested

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,7 +30,7 @@ doc/api/
 .idea/
 
 # Visual Studio Code related
-.vscode/
+# .vscode/
```
Comment (svrnm) [inline .gitignore:33]: why did you change that? .vscode (or any other editor specific files) should not be part of the code repository.

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -22,24 +22,23 @@ if (flutterVersionName == null) {
...
 
-    lintOptions {
-        disable 'InvalidPackage'
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
     }
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.appdynamics.appdynamics_mobilesdk_example"
```
Comment (svrnm) [inline example/android/app/build.gradle:37]: why is this renamed?

**Comment by svrnm** [comment]
@jWinterDay there is lots of changes on different places I do not understand, can you give me a little bit more background on why so many files have changed that drastically? thanks.

File: example/android/app/src/main/assets/config.sample.properties [hunk 3]
```diff
@@ -1 +0,0 @@
-APPDYNAMICS_EUM_KEY=NO_KEY_SET
```
[the remark this answers is not in the document]
Comment (jWinterDay) [reply example/android/app/src/main/assets/config.sample.properties:1]: I regenerated example project. It was replaced. I can restore your example if you want

File: .gitignore (hunk 1 above)
Comment (jWinterDay) [reply .gitignore:33]: Ok, I used my project settings (autoformat code for 120 length). I'll reformat to 80 length and remove this file
